Snakefile

```diff
@@ -153,6 +156,10 @@ rule merge_input:
         dataset_dict = get_dataset(_config.config.datasets, wildcards.dataset)
         runner.merge_input(dataset_dict, output.dataset_file)
 
+# TODO: add a merge input for gold standard data?
```

Comment:
Yes, we will need an equivalent of merge input to gather all of the text files per gold standard dataset and create the pickled gold standard object.